docs/sphinx/ome-tiff/specification.rst

```diff
@@ -434,3 +438,96 @@ The master file containing the full OME-XML metadata should be either an
 OME-XML companion file with the extension :file:`.companion.ome` or a master
 OME-TIFF file containing the full metadata (see :ref:`multifile_samples` for
 representative samples).
+
+Pyramidal OME-TIFF
+------------------
+
+.. versionadded:: 6.0.0
+
+The OME-TIFF specification supports multi-resolution images or pyramidal
+images where individual planes are stored at different levels of resolution.
+The intermediate downsampled image planes are called pyramidal levels,
+sub-resolution image planes or sub-resolutions.
+
+Supported resolutions
+^^^^^^^^^^^^^^^^^^^^^
+
+OME-TIFF planes can be reduced alongside the X and Y dimensions. Each pyramidal
+level must be a downsampling of the full-resolution plane in the X and Y
+dimensions and the resolution should stay unchanged in the other dimensions.
+The downsampling factor should be an integer value, identical alongside the X
+and the Y dimensions and stay the same between each consecutive pyramidal
+level.
+
+The following table below shows two examples of pyramidal level dimensions
+using typical downsampling factors:
+
+.. list-table::
+  :header-rows: 1
+
+  -  *
+     * Example 1
+     * Example 2
+
+  -  * Downsampling factor
+     * 3
+     * 4
+
+  -  * Level 0 (full-resolution)
+     * 9234 × 6075 × 1 × 1 × 10
+     * 38912 × 25600 × 1 × 3 × 1
+
+  -  * Level 1
+     * 3078 × 2025 × 1 × 1 × 10
+     * 9728 × 6400 × 1 × 3 × 1
+
+  -  * Level 2
+     * 1026 × 675 × 1 × 1 × 10
+     * 2432 × 1600 × 1 × 3 × 1
+
+  -  * Level 3
+     * 342 × 225 × 1 × 1 × 10
+     * 608 × 400 × 1 × 3 × 1
+
+  -  * Level 4
+     * 114 × 74 × 1 × 1 × 10
+     * 152 × 100 × 1 × 3 × 1
+
+  -  * Level 5
+     * 38 × 25 × 1 × 1 × 10
+     * 38 × 25 × 1 × 3 × 1
+
+
+Storage
+^^^^^^^
+
+Full-resolution image planes must reamin stored as described above using a valid TIFF IFD and referenced from the OME-XML metadata using the
+:ref:`TiffData <tiffdata>` element.
+
+Each sub-resolution must be stored in a valid IFD of the same TIFF file as the full-resolution image plane. Additionally:
```

Comment:
`in a valid IFD of the same TIFF file` should be `in the same IFD`.  The validity of the IFD, and the TIFF file in question are implicit; IFDs can't be split between files.